examples/subscription/reactivate-subscription.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This example shows you how to reactivate a subscription
+ */
+
+include __DIR__ . '/../../vendor/autoload.php';
+
+try {
+    $sharedSecret = getenv('VENDO_SHARED_SECRET', true)?:'Your_Vendo_Shared_Secret__get_it_from_us';
+
+    $reactivate = new \VendoSdk\Subscription\ReactivateSubscription($sharedSecret);
+    $reactivate->setMerchantId(getenv('VENDO_MERCHANT_ID',  true) ?: 'Your_vendo_merchant_id');//Your Vendo Merchant ID
+    $reactivate->setSubscriptionId(72537045);//The Vendo Subscription ID that you want to reactivate.
+
+    $response = $reactivate->getRequest();
+
+    echo "\n\nRESULT BELOW\n";
+    if ($response->getResponseCode() == \VendoSdk\Vendo::SUBSCRIPTION_CANCEL_RESPONSE_CODE_OK) {
```

Comment:
The constant is wrong you should create a new one for the correct reactivate responded code OK